### PR TITLE
Update turabian-author-date.csl

### PR DESCRIPTION
**Pull request by dstark**
Corrects the citation output in three cases:

1) inclusion of seasons of publication in journal article reference list items,
2) in-text citations for reprinted books, and 
3) reference list items for reprinted books.

### CODE REVIEW

**Review by adam3smith — CHANGES_REQUESTED** [review]
Thanks! A couple of small requests

**Review by dstark — COMMENTED** [review]
Revised file with 3 requested changes from @adam3smith.

File: turabian-author-date.csl [hunk 1]
```diff
@@ -445,7 +448,12 @@
       </else>
     </choose>
   </macro>
+  <macro name="original-publication-date">
+    <!-- Adds support for placing the original publication date at the end of the reference list entry per Turabian, 9th ed., §19.1.4. -->
+    <date variable="original-date" form="text" date-parts="year" prefix=" (Orig. pub. " suffix=".)"/>
+  </macro>
   <macro name="date-in-text">
+    <!-- Omits the original publication date in the in-text citation per Turabian, 9th ed., §19.1.4. -->
```
Comment (adam3smith) [inline turabian-author-date.csl:456]: This comment is no longer relevant/correct, please delete

**Review by adam3smith — CHANGES_REQUESTED** [review]
One comment left to delete. Than!

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**